src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/jpa/entity/RecommendationEntity.kt

```diff
@@ -1,26 +1,46 @@
 package uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity
 
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType
 import org.hibernate.annotations.Type
+import org.hibernate.annotations.TypeDef
 import java.io.Serializable
 import javax.persistence.Column
 import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
 import javax.persistence.Table
 
 @Entity
 @Table(name = "recommendations")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType::class)
 data class RecommendationEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  open var id: Long? = null,
+
   @Type(type = "jsonb")
   @Column(columnDefinition = "jsonb")
   var data: RecommendationModel
-) : EntityWithId()
+)
 
 data class RecommendationModel(
-  var name: String? = null,
   val crn: String?,
   var recommendation: Recommendation? = null,
-  var alternateActions: String? = null
+  var active: Active? = null,
```

Comment:
Should this not be called 'Status' instead of 'Active'?

Reply:
Done :)